package.json

```diff
@@ -41,7 +41,7 @@
   },
   "scripts": {
     "dev": "PORT=3001 craco start",
-    "dev:remote": "sudo HTTPS=true PORT=443 IS_REMOTE_ENV=true craco start",
+    "dev:remote": "HTTPS=true PORT=443 IS_REMOTE_ENV=true craco start",
```

Comment:
`sudo` is needed on macOS, maybe it would be good in the future to add explicit instruction how to run this task on mac

Reply:
yes, accidentally pushed it without. Let's keep it in there since most of you guys are using macOS.